benchmark.cfg.example

```diff
@@ -20,7 +20,7 @@ query_levels=[1, 5,10,15,20]
 threads=8
 mode=benchmark
 os=linux
-server_host=127.0.0.1
+server_host=TFB-server
```

Comment:
There was once, long ago, an implementation similar to this; we removed it because some frameworks could not resolve names for reasons we could not explain. Hopefully, this is no longer the case....